modules/planning/scenarios/side_pass/side_pass_scenario.h

```diff
@@ -61,6 +61,12 @@ class SidePassScenario : public Scenario {
 
   bool IsFarFromDestination(const Frame& frame);
 
+  bool IsWithinSidePassingSpeedADC(const Frame& frame);
+
+  bool IsSidePassableObstacle(
+    const Frame& frame, const ReferenceLineInfo& reference_line_info,
+    const std::string& blocking_obstacle_id);
```

Comment:
`bool function() const;` if it doesn't change the instance.

Reply:
Fixed this and other similar occurrences.